src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -0,0 +1,189 @@
+from __future__ import absolute_import, division, print_function
+
+import functools
+import six
+
+from django.conf import settings
+
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.group import StreamGroupSerializerSnuba
+from sentry.constants import DEFAULT_SORT_OPTION
+from sentry.models import Environment, Group, GroupStatus, Project
+from sentry.models.group import looks_like_short_id
+from sentry.search.snuba.backend import SnubaSearchBackend
+from sentry.search.utils import InvalidQuery, parse_query
+from sentry.utils.cursors import Cursor
+
+
+ERR_INVALID_STATS_PERIOD = "Invalid stats_period. Valid choices are '', '24h', and '14d'"
+
+
+search = SnubaSearchBackend(**settings.SENTRY_SEARCH_OPTIONS)
+
+
+class ValidationError(Exception):
+    pass
+
+
+class OrganizationGroupIndexEndpoint(OrganizationEventsEndpointBase):
```

Comment:
Is it possible to abstract the logic here and share with the project version of the endpoint?

Reply:
moved some stuff into shared helpers